tests/importance_tests/test_init.py

```diff
@@ -94,7 +97,6 @@ def objective(trial: Trial) -> float:
             assert isinstance(importance, float)
             assert importance <= prev_importance
             prev_importance = importance
-        assert math.isclose(1.0, sum(i for i in param_importance.values()), abs_tol=1e-5)
```

Comment:
Is it necessary to remove these assertions even though `parametrize_evaluator` does not contain ShapleyImportanceEvaluator`?

Reply:
If you decide to remove the specification that importance values should add up to 1, I think we should also remove them from existing tests.

Reply:
But maybe this could be left for another PR.